frontend/app/src/composables/blockchain/index.ts

```diff
@@ -269,12 +269,14 @@ export function useBlockchains() {
     }
 
     const registeredAddresses: string[] = [];
-    const failedPayload: AccountPayload[] = [];
+    const failedPayload: (AccountPayload | XpubAccountPayload)[] = [];
```

Comment:
can you check the changes from the account's branch for this and pull them over?